[2022-01-19 20:01:30,241] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.database_processing_media_prepare manual__2022-01-19T20:01:15.496142+00:00 [queued]>
[2022-01-19 20:01:30,258] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.database_processing_media_prepare manual__2022-01-19T20:01:15.496142+00:00 [queued]>
[2022-01-19 20:01:30,259] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-01-19 20:01:30,259] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-01-19 20:01:30,260] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-01-19 20:01:30,280] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): database_processing_media_prepare> on 2022-01-19 20:01:15.496142+00:00
[2022-01-19 20:01:30,287] {standard_task_runner.py:52} INFO - Started process 177 to run task
[2022-01-19 20:01:30,292] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'database_processing_media_prepare', 'manual__2022-01-19T20:01:15.496142+00:00', '--job-id', '2808', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpqsvi37dj', '--error-file', '/tmp/tmpzqtdiqvl']
[2022-01-19 20:01:30,295] {standard_task_runner.py:77} INFO - Job 2808: Subtask database_processing_media_prepare
[2022-01-19 20:01:30,362] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.database_processing_media_prepare manual__2022-01-19T20:01:15.496142+00:00 [running]> on host 218df5ddc363
[2022-01-19 20:01:30,450] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=database_processing_media_prepare
AIRFLOW_CTX_EXECUTION_DATE=2022-01-19T20:01:15.496142+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-19T20:01:15.496142+00:00
[2022-01-19 20:01:30,453] {python.py:152} INFO - Done. Returned value was: None
[2022-01-19 20:01:30,478] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=database_processing_media_prepare, execution_date=20220119T200115, start_date=20220119T200130, end_date=20220119T200130
[2022-01-19 20:01:30,508] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-19 20:01:30,567] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
